samples/Playground/Directory.Packages.props

```diff
@@ -29,16 +29,16 @@
 		<PackageVersion Include="Uno.Toolkit.UI.Material" Version="2.4.0-dev.85" />
 		<PackageVersion Include="Uno.Toolkit.WinUI" Version="2.4.0-dev.85" />
 		<PackageVersion Include="Uno.Toolkit.WinUI.Material" Version="2.4.0-dev.85" />
-		<PackageVersion Include="Uno.UI" Version="4.6.0-dev.947" />
-		<PackageVersion Include="Uno.UI.Adapter.Microsoft.Extensions.Logging" Version="4.6.0-dev.947" />
-		<PackageVersion Include="Uno.UI.RemoteControl" Version="4.6.0-dev.947" />
-		<PackageVersion Include="Uno.UI.Skia.Gtk" Version="4.6.0-dev.947" />
-		<PackageVersion Include="Uno.UI.Skia.Tizen" Version="4.6.0-dev.947" />
-		<PackageVersion Include="Uno.UI.Skia.Wpf" Version="4.6.0-dev.947" />
-		<PackageVersion Include="Uno.UI.WebAssembly" Version="4.6.0-dev.947" />
+		<PackageVersion Include="Uno.UI" Version="4.6.18" />
```

Comment:
You'll need to use the .19 to avoid winappsdk-specific issues

Reply:
yep, will be updating to that when I update toolkit